src/styles/_gravity-overrides.scss

```diff
@@ -7,23 +7,35 @@ main {
   margin-top: 0;
 }
 
+img {
+  width: 100%;
```

Comment:
isn't there a max-width already present and do we even have any images (there all SVGs no?)

Reply:
This was to fix an issue with img's but yes no longer needed now